cmd/shortener/main.go

```diff
@@ -160,3 +166,93 @@ func run(cfg *config.App, log *zap.Logger) error {
 		}
 	}
 }
+
+func baseInit(cfg *config.App, log *zap.Logger) (*usecase.Usecase, usecase.UserAddRepository, error) {
+	var store usecase.Store
+	var err error
+	var pingable usecase.Pingable
+	var jwtUserRep usecase.UserAddRepository
```

Comment:
Можно сделать объявление переменных немного более элегантным.
```suggestion
	var (
	         store usecase.Store
	         err error
                 pingable usecase.Pingable
	         jwtUserRep usecase.UserAddRepository
	 )
```